docs/02.API-REFERENCE.md

```diff
@@ -757,6 +757,42 @@ jerry_eval (const jerry_char_t *source_p,
 - [jerry_create_external_function](#jerry_create_external_function)
 - [jerry_external_handler_t](#jerry_external_handler_t)
 
+## jerry_run_all_enqueued_jobs
+
+**Summary**
+
+Run enqueued Promise jobs until the first thrown error or until all get executed.
+
```

Comment:
Sorry, I just saw your last comment. The note for this should be added in this PR. The missing notes from other API functions can go in a new PR, because they are not related to this patch.